apps/web/lib/orgMigration.ts

```diff
@@ -692,20 +725,22 @@ async function dbRemoveTeamFromOrg({ teamId, targetOrgId }: { teamId: number; ta
     });
   }
 
-  if (team.parentId !== targetOrgId) {
-    log.warn(`Team ${teamId} is not part of org ${targetOrgId}. Not updating`);
-    return {
-      slug: team.slug,
-    };
-  }
```

Comment:
No need to unnecessarily stop the revert as it can possibly allow fixing things by running revert again .